public/app/core/components/Login/LoginCtrl.tsx

```diff
@@ -111,6 +130,49 @@ export class LoginCtrl extends PureComponent<Props, State> {
       });
   };
 
+  passwordlessStart = (formModel: PasswordlessFormModel) => {
+    this.setState({
+      loginErrorMessage: undefined,
+      isLoggingIn: true,
+    });
+
+    getBackendSrv()
+      .post<AuthNRedirectDTO>('/login/passwordless/start', formModel, { showErrorAlert: false })
```

Comment:
is this an API endpoint on a non-`/api/` route? Seems kinda wonky tbh

Reply:
good point, this could/should be an api route